yarn-project/end-to-end/src/guides/dapp_testing.test.ts

```diff
@@ -157,40 +185,61 @@ describe('guides/dapp/testing', () => {
 
       it('checks public storage', async () => {
         // docs:start:public-storage
-        await nativeToken.methods.owner_mint_pub(owner.getAddress(), 100n).send().wait();
-        const ownerPublicBalanceSlot = cheats.aztec.computeSlotInMap(4n, owner.getAddress());
-        const balance = await rpc.getPublicStorageAt(nativeToken.address, ownerPublicBalanceSlot);
+        await token.methods.mint_public({ address: owner.getAddress() }, 100n).send().wait();
+        const ownerPublicBalanceSlot = cheats.aztec.computeSlotInMap(6n, owner.getAddress());
+        const balance = await rpc.getPublicStorageAt(token.address, ownerPublicBalanceSlot);
         expect(toBigIntBE(balance!)).toEqual(100n);
         // docs:end:public-storage
       });
 
-      it('checks unencrypted logs', async () => {
+      it('checks unencrypted logs, [Kinda broken with current implementation]', async () => {
```

Comment:
issue to fix?

Reply:
Created https://github.com/AztecProtocol/aztec-packages/issues/2306 for it, will update